src/scene-entry-manager.js

```diff
@@ -417,24 +421,22 @@ export default class SceneEntryManager {
 
   _setupCamera = () => {
     this.scene.addEventListener("action_toggle_camera", () => {
-      if (!this.hubChannel.can("spawn_camera")) return;
-      const myCamera = this.scene.systems["camera-tools"].getMyCamera();
-
-      if (myCamera) {
-        myCamera.parentNode.removeChild(myCamera);
+      const myCam = anyEntityWith(APP.world, MyCameraTool);
```

Comment:
Note the concept of what "my camera" is was intentionally simplified. Previously if someone grabbed a camera you spawned we (for some purposes) considered it to be "their" camera (though it would still get deleted if you left the room. Now you can only spawn 1 camera, and that one is always "yours". To remove it you click the button (or press c) again. The camera has no object menu.